pyteal/ast/abi/array_base_test.py

```diff
@@ -1,48 +1,60 @@
 from typing import List, cast
 import pytest
 
-from ... import *
-
-options = CompileOptions(version=5)
-
-STATIC_TYPES: List[abi.TypeSpec] = [
-    abi.BoolTypeSpec(),
-    abi.Uint8TypeSpec(),
-    abi.Uint16TypeSpec(),
-    abi.Uint32TypeSpec(),
-    abi.Uint64TypeSpec(),
-    abi.TupleTypeSpec(),
-    abi.TupleTypeSpec(abi.BoolTypeSpec(), abi.BoolTypeSpec(), abi.Uint64TypeSpec()),
-    abi.StaticArrayTypeSpec(abi.BoolTypeSpec(), 10),
-    abi.StaticArrayTypeSpec(abi.Uint8TypeSpec(), 10),
-    abi.StaticArrayTypeSpec(abi.Uint16TypeSpec(), 10),
-    abi.StaticArrayTypeSpec(abi.Uint32TypeSpec(), 10),
-    abi.StaticArrayTypeSpec(abi.Uint64TypeSpec(), 10),
-    abi.StaticArrayTypeSpec(
-        abi.TupleTypeSpec(abi.BoolTypeSpec(), abi.BoolTypeSpec(), abi.Uint64TypeSpec()),
+import pyteal as pt
+
+options = pt.CompileOptions(version=5)
+
+STATIC_TYPES: List[pt.abi.TypeSpec] = [
+    pt.abi.BoolTypeSpec(),
```

Comment:
👍 

`pt.abi...` caught me by surprise, but is certainly in the spirit of incoming [CONTRIBUTING.md](https://github.com/algorand/pyteal/blob/d22a93ac9b220c52ab60a21e6292d534283941f1/CONTRIBUTING.md)